docusaurus/docs/Android/01-basics/03-getting-started.md

```diff
@@ -0,0 +1,108 @@
+# Getting started
```

Comment:
Some of the content on this page is new, most is just from the page previously named "Initializing the SDK"